project-metrics/metrics_service/metrics/base.py

```diff
@@ -32,29 +40,81 @@ def label(self) -> Text:
   @property
   def formatted_result(self) -> Text:
     """The formatted result value (ex. 3w, 80%, 100PRs)."""
-    return self._format_value(self._result.value) if self._result else '?'
-
-  @property
-  def result(self) -> Optional[models.MetricResult]:
-    """The result of a computation of the metric."""
-    return self._result
+    return self._format_value(self.result.value) if self.result else '?'
 
   @property
   def score(self) -> models.MetricScore:
     """The 0-4 score for the metric."""
     return self._score_value(
-        self._result.value) if self._result else models.MetricScore.UNKNOWN
+        self.result.value) if self.result else models.MetricScore.UNKNOWN
+
+
+class Metric(MetricDisplay):
+  """Abstract base class for health metrics."""
+
+  __metaclass__ = abc.ABCMeta
+  __active_metrics = {}
```

Comment:
nit: just one underscore (here and in the method below) - two underscores have a complicated meaning in Python

Reply:
Done.